backend/src/main/java/com/ody/mate/service/MateService.java

```diff
@@ -17,20 +19,31 @@
 public class MateService {
 
     private final MateRepository mateRepository;
+    private final NotificationService notificationService;
 
     @Transactional
+    public MeetingSaveResponse saveAndSendNotifications(
```

Comment:
[제안]

saveAndSendNotifications도 구현이 모두 반영된 네이밍이라 생각되어 다음과 같은 네이밍 제안합니다!
```suggestion
    public MeetingSaveResponse entry(
```